lib/pages/home/plan_map/plan_empty.dart

```diff
@@ -0,0 +1,93 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_map/flutter_map.dart';
+import 'package:latlong/latlong.dart';
+import 'package:trufi_core/widgets/map/buttons/map_type_button.dart';
+import 'package:trufi_core/widgets/map/buttons/your_location_button.dart';
+import 'package:trufi_core/widgets/map/map_copyright.dart';
+import 'package:trufi_core/widgets/map/trufi_map_controller.dart';
+import 'package:trufi_core/widgets/map/trufi_map.dart';
+
+import '../../../trufi_app.dart';
+
+const double customOverlayWidgetMargin = 80;
+
+class PlanEmptyPage extends StatefulWidget {
+  const PlanEmptyPage(
+      {this.initialPosition,
+      this.onLongPress,
+      this.customOverlayWidget,
+      this.customBetweenFabWidget,
+      Key key})
+      : super(key: key);
+
+  final LatLng initialPosition;
+  final LongPressCallback onLongPress;
+  final LocaleWidgetBuilder customOverlayWidget;
+  final WidgetBuilder customBetweenFabWidget;
+
+  @override
+  PlanEmptyPageState createState() => PlanEmptyPageState();
+}
+
+class PlanEmptyPageState extends State<PlanEmptyPage>
+    with TickerProviderStateMixin {
+  final _trufiMapController = TrufiMapController();
+
+  @override
+  Widget build(BuildContext context) {
+    final Locale locale = Localizations.localeOf(context);
+    return Stack(
+      children: <Widget>[
+        TrufiMap(
+          key: const ValueKey("PlanEmptyMap"),
+          controller: _trufiMapController,
+          onLongPress: widget.onLongPress,
+          layerOptionsBuilder: (context) {
+            return <LayerOptions>[];
```

Comment:
I guess we do not need a type here because it is inferred from the layerOptionsBuilder
```suggestion
            return [];
```